src/main/java/uk/gov/hmcts/darts/event/controller/DarNotifyController.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.darts.event.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.darts.event.model.DarNotifyEvent;
+import uk.gov.hmcts.darts.event.service.DarNotifyEventService;
+
+@Slf4j
+@RequiredArgsConstructor
+@RestController
+public class DarNotifyController {
+
+    private final DarNotifyEventService darNotifyEventService;
+
+    @PostMapping(
+        value = "/events/dar-notify",
+        consumes = MediaType.APPLICATION_JSON_VALUE,
+        produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Void> darNotify(@RequestBody DarNotifyEvent darNotifyEvent) {
+        log.info("Received {}", darNotifyEvent);
```

Comment:
Will this spit out the whole dar event object? Do we really want to be doing that? Should probably just be an id? Can you change it to a more tailored message as well e.g. "Received a dar notify event for ......"

Reply:
I'll just remove this log for now, running `DarNotifyControllerTest#shouldSendDarNotifyEventSoapAction` would spit out something like this in the application log

 `2023-06-23T12:41:18.266 INFO  [Test worker] u.g.h.darts.event.controller.DarNotifyController Received DarNotifyEvent(notificationUrl=null, notificationType=3, timestamp=2023-06-19T14:52:40.637Z, courthouse=Test Court, courtroom=1, caseNumbers=[A123456])
`